build-data/paper.at

```diff
@@ -233,3 +233,6 @@ public net.minecraft.world.level.entity.PersistentEntitySectionManager sectionSt
 
 # Optimize light engine
 public-f net.minecraft.world.level.chunk.DataLayer
+
+# Expose tracked entities
+public-f net.minecraft.server.level.ChunkMap.TrackedEntity range
```

Comment:
Ack, I misread. Should be `+f` to retain final I believe?
@kennytv can probably correct me

Reply:
Can just omit the -f

Reply:
`+f` is not present in the file. Would `public` not retain the `final` if `-f` would remove it?